Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Materials/HolographicBackPlateHubTray.mat

```diff
@@ -0,0 +1,189 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 6
+  m_ObjectHideFlags: 0
```

Comment:
Maybe put this under the Examples/ExampleHub/Materials?

Reply:
Actually, I realized that this is no longer used. Deleted the asset.